src/OctoshiftCLI.Tests/GithubApiTests.cs

```diff
@@ -990,7 +990,7 @@ public async Task GetTeamSlug_Returns_The_Team_Slug()
                     description = "A great team.",
                     privacy = "closed",
                     permission = "admin",
-                    members_url = "https://api.github.com/teams/1/members/membber",
+                    members_url = "https://api.github.com/teams/1/members/member",
```

Comment:
If this is intentional, might I suggest:

```suggestion
                    members_url = "https://api.github.com/teams/1/members/member_",
```